### DISCUSSION TITLE
Install latest version on Mac OS X

### DISCUSSION THREAD

**Discussion by TCke83** [mopidy/mopidy-spotify#395] [Q&A]
I want to replicate my setup of mopidy (version 4) on a Mac mini, i have some trouble with my USB audio device on a raspberry pi, i did some tests and those problems seem to be abscent on Mac OS X. I've succeeded to install mopidy (and it's dependencies) using homebrew. But now i need you get libspotify, mopidy & mopidy-spotify and build them from the source. Does somebody have some hints on how to do this?

**Comment by kingosticks** [comment]
We stopped using libspotify some time ago. Mopidy-Spotify is now a pure python extension and you should install the latest pre-release version from GitHub, as stated in the readme. You'll also need gst-plugin-spotify which can be either built from source or there are binaries for some platforms available. This is also explained in the readme.
